src/backend/controllers/tankController.js

```diff
@@ -229,7 +229,7 @@ exports.unfavoriteTankTeam = async (req: Request, res: Response) => {
 	}
 
 	// Find the user and set favoriteTank to null.
-	const user = await User.findById(req.user.id, 'wager money favoriteTanks');
+	const user = await User.findById(req.user.id, 'wager3v3 money favoriteTanks');
 
```

Comment:
Bleh the copy/pasta likely is the culprit here

Reply:
Yeah I made this I'm pretty sure, so mb.